platform/android/CHANGELOG.md

```diff
@@ -4,8 +4,11 @@ Mapbox welcomes participation and contributions from everyone.  If you'd like to
 
 ## master
```

Comment:
per voice, this is resolved. instead we will start each release's changelog with:

[Changes](https://github.com/mapbox/mapbox-gl-native/compare/android-v8.2.0-alpha.2...android-v8.2.0-alpha.3) since [Mapbox Maps SDK for Android v8.2.0-alpha.3](https://github.com/mapbox/mapbox-gl-native/releases/tag/android-v8.2.0-alpha.3):

@langsmith